README.md

```diff
@@ -74,10 +76,28 @@ Providing your own icons is crucial as many of our components use an `Icon` inte
     <limel-config config={{iconPath: '/my/parent/path/'}} />
     ```
 
+---
+
+## Using components marked as `@beta`
+
+Some components in Lime Elements are marked as `@beta`. These components are experimental and may still be in development. They could contain bugs and their APIs and interfaces might change without notice. This means that, while you can use these components, you should be aware that they are subject to changes and may not yet be fully stable.
+
+Key Points:
+
+1. **Experimental Status:** Components marked as `@beta` are not final and may evolve based on feedback and further development.
+1. **Potential Issues:** Be prepared for potential bugs or incomplete features.
+1. **Interface Changes:** We may alter their interfaces without formally documenting these changes in the change log. This allows us to refine the components based on real-world usage and feedback.
+
+Use `@beta` components with caution and at your own risk. They are available for early testing and feedback, helping us improve the library for future releases.
```

Comment:
I don't think this explanation of the `@beta` tag is good enough. What needs to be clear is that we may make _breaking changes_ to a component marked as `@beta`, without bumping the major version number.

No components are final, all components may evolve based on feedback and further development.

Reply:
what about this one 👇